apps/web/README.md

```
@@ -0,0 +1,117 @@
+
+### Linting
+
+Run ESLint:
+
+```bash
+pnpm lint
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)